doc/source/how-to/code/build.yml

```diff
@@ -0,0 +1,36 @@
+name: CI
+on:
+  pull_request:
+  push:
+    tags:
+      - "*"
+    branches:
+      - main
+
+jobs:
+  build:
+    name: Build
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout project
+        uses: actions/checkout@v2
+
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.9
+
```

Comment:
We should really start having an env variable in our CI like MAIN_PYTHON_VERSION. And it should hold the version we are using for dev. I see plenty of times this:
```yml
        with:
          python-version: 3.9
```
And we could do something like
```yml
        with:
          python-version: ${{ env.MAIN_PYTHON_VERSION }}
```

I already did this in PyAnsys Tools Report if you want to take this as a reference. It is not the same changing once the main python version than changing it N times across all jobs/stages